test/main.cpp

```diff
@@ -30,7 +34,51 @@
 
 using namespace std;
 
+static void set_filters_for_backend() {
+  std::string backend = "GENERIC";
```

Comment:
Default to "CPU".

Reply:
So, do we want to create all flavors of the cpp_tests_list_<XXX>.txt file in this PR?